### ISSUE TITLE
tf.softmax() and tf.logSoftmax() should call into a kernel instead of composing high-level ops.

### ISSUE THREAD

**Issue by nsthorat** [tensorflow/tfjs#884]
Currently we write softmax / logSoftmax as a combination of high-level ops. We should move these into kernels, like done in TensorFlow python.

For the WebGL backend, we can still compose this of higher-level ops, but in Node.js we'll be able to call the C++ kernel for softmax.

**Comment by rthadur** [comment]
I guess the related PRs [PR1](https://github.com/tensorflow/tfjs/pull/2767) and [PR2](https://github.com/tensorflow/tfjs/pull/2289) are merged here , so closing this issue, thank you
